### ISSUE TITLE
No activity from @E.Beef for over a year?!? Where do I go?

### ISSUE THREAD

**Issue by RogueLogix** [erogenousbeef/BigReactors#583]
TLDR: Go [here](https://minecraft.curseforge.com/projects/extreme-reactors) for downloads and [here](https://github.com/ZeroNoRyouki/BigReactors) for the github.

As you may know Mr. Beef has been inactive for over a year, as of writing this. because at this time 1.7.10 is being phased out of modded minecraft with newer modpacks being built (Sky Factory 3 for example) Big Reactors needed to be updated or die. Thanks to @ZeroNoRyouki we have 1.9.4, 1.10.2, 1.11.2, and 1.12 versions of Big Reactors. You can find all information about the port [Extreme Reactors](https://minecraft.curseforge.com/projects/extreme-reactors) at that link. At time of writing Zero is actively updating ER and fixing issues found with the mod.

For modders.
If you love the way that the multiblocks in Big Reactors work then your not alone. As per my request Zero created a separate API for the multiblocks that you can use to make whatever you want. This takes most of the hassle out of actually making a multiblock structure. you can find a tutorial on the API over at Zero's [website](http://zerono.it/).
